src/screens/RegisterEnsSheet.js

```diff
@@ -25,6 +26,54 @@ const Container = styled.View`
   flex: 1;
 `;
 
+const AvailabilityIndicator = props => {
```

Comment:
I think it's convention to spread props?
```suggestion
const AvailabilityIndicator = ({ isRegistered }) => {
```